modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -24,6 +24,7 @@ Perform the migration only when an interruption in service is acceptable.
 * A recent backup of the etcd database is available.
 * A reboot can be triggered manually for each node.
 * The cluster is in a known good state, without any errors.
+* For clusters on AWS, an AWS security group rule in place to allow `udp` packets on the 6081 port for all nodes.
```

Comment:
Is this for user-provisioned infrastructure only? I see the link is for cloudformation templates for user-infra.

Reply:
for IPI from 4.2. the port already be opened in security group of aws when cluster deploying. so usually customer do not need to do that by manual.  unless the cluster was upgraded from 4.1.
for UPI cluster, customer need to check the port in security group of aws to make sure 6081 is opened

Reply:
Does this only happen in AWS? Would it also be an issue in GCP and Azure?

Reply:
@pliurh this issue should be only happen on cluster upgraded from 4.1 version.  for GCP . if the cluster is also upgraded from 4.1.  I guess also this issue.  see details comment https://issues.redhat.com/browse/OCPBUGS-16200

Reply:
How about update the statement to be "On all cloud platforms, a security group rule must be in place to allow UDP packets on port `6081` for all nodes after software upgrading and before SDN to OVN-K migration"?